### DISCUSSION TITLE
When is it expected that Ariakit v1 arrives, and semver starts to be followed?

### DISCUSSION THREAD

**Discussion by gnapse** [ariakit/ariakit#4113] [Questions] [answered]
It'd be convenient to set Ariakit as a peer dependency, but to do so, it'd be convenient to know that semver is followed, so we can set a specific range of major versions supported as peer dep.

**Comment by diegohaz** [comment, marked as answer]
v1 will be released with the launch of the website ariakit.org. The library is technically stable, it's just not v1 for marketing reasons. See [This isn’t an official announcement yet](https://newsletter.ariakit.org/i/133152153/this-isnt-an-official-announcement-yet).

-  Patches and minor changes bump the Z version in X.Y.Z.
-  Major changes bump the Y version.

If you want to add `@ariakit/react` as a peer dependency, you can use the `~` symbol to accept only patch versions (since minor versions represent breaking changes in v0):

```json
"@ariakit/react": "~0.4.0"
```

**Reply by gnapse** [reply]
Thanks!

**Reply by mayank99** [reply]
Small clarification: `^0.4.0` and `~0.4.0` are effectively equivalent. Before a package reaches 1.0, `^` only includes patch versions in the range. Try it [here](https://semver.npmjs.com/) and [here](https://semver.otterlord.dev/?package=@ariakit/react&range=^0.3.0).

**Comment by mayank99** [comment]
Is there anything we can do to help release 1.0? It might be good to have a tracking issue/milestone for 1.0, so that people can contribute and help reach the finish line faster.
